### ISSUE TITLE
Super laggy

### ISSUE THREAD

**Issue by Fooks-Applications** [DarkGamerYT/Bedrock-Editor-Extension#5]
when using the tools my frames drop A LOT and in the newest update, specific tools just stop working because of how low the fps gets. I'm running a Ryzen 7 5800H with an RTX 3060. It shouldn't lag so badly. Other than that, awesome editor extension :)

**Comment by DarkGamerYT** [comment]
Can you specify which tools are causing your game to lag?

**Comment by Fooks-Applications** [comment]
It's all of them, it's really weird. When I have the addon enabled it's
super laggy. Also, when using a big tool like a sphere, it doesn't generate
fully and then cuts in half or something

**Comment by DarkGamerYT** [comment]
The reason the sphere doesn’t generate fully is because of the watchdog system that Mojang added to the Creator API

Will look more into the lag

**Comment by Fooks-Applications** [comment]
Cool, thanks! What's your email? I have been watching you a Twitter for a
long time but I don't have a Twitter account so I can never talk to you

**Comment by Fooks-Applications** [comment]
No, I don't have discord either. You have an email or number or something
